### ISSUE TITLE
Incorrect TPS

### ISSUE THREAD

**Issue by AsVaidas** [VolmitSoftware/React#326]
![sketch 5](https://user-images.githubusercontent.com/5215112/39408655-3bff76ea-4be2-11e8-9f91-e45fb0c36a44.png)

**Comment by lucas2107** [comment]
React display current TPS wheh your doing /tps command, essentials display average tps.

Probably your TPS dropped when you ran the command.

**Comment by cyberpwnn** [comment]
Could you show a few additional things?

### When TPS values do not match
* Use `/re map` and look at the TICK graph (first graph) and screenshot it
* Use `/re dump` and send the link
* A Timings report would be nice, but i really only need 5-10 seconds of sample time

**Comment by AsVaidas** [comment]
@lucas2107 
No, the server was stable, the tps is always 10-15, when timings show 20 tps

@cyberpwnn 
The issue appears with 30-40 players, so I will wait for the middle of the day and send the data
P.S. /re dump doesn't work (doesn't give any information or link, only the react help page) version: 6.4.3

**Comment by AsVaidas** [comment]
P.S.S.
![image](https://user-images.githubusercontent.com/5215112/39431716-329926ce-4c9a-11e8-84e6-506882fa7118.png)
https://timings.aikar.co/?id=4b23728802164f8bbf4a2ea470011cd2

**Comment by AsVaidas** [comment]
@cyberpwnn 
As I said, /re dump doesn't work

![tpsincorrect](https://user-images.githubusercontent.com/5215112/39441724-863e2fbe-4cb7-11e8-937a-3a3296cb15c9.png)
![tpsincorrect2](https://user-images.githubusercontent.com/5215112/39441725-87c615f4-4cb7-11e8-9142-e9e6b41533df.png)
https://timings.aikar.co/?id=c71bab1e067b4b778d896a4ec4356560

**Comment by cyberpwnn** [comment]
Use /re a dump

**Comment by cyberpwnn** [comment]
I may have fixed it in 6.4.5
* TPS Average radius increased from 6 to 10 ticks
* Added an additional TPS probe in a different method which dynamically switches over if it is reporting a higher value than the default tps reading

**Comment by cyberpwnn** [comment]
It seems it's still causing issues, working on it.
